api/control/v1alpha1/types.go

```diff
@@ -443,6 +457,42 @@ const (
 // Type checking errors are posted to the status.Errors of the Binding and BindingPolicy.
 type Expression string
 
+// Augmentation defines what is implicitly queryable in the CEL expressions.
+// This means that while the listed fields are not part of the object's schema,
+// they can be referred to in the CEL expressions.
+//
+// Besides for Augmentations, the other objects queryable in CEL expressions are:
+//
+//  1. The workload object which resides in the WDS, as `obj`;
+//
+//  2. The following subresources of the propagated object at a WEC, as `returned`:
+//
+//     - .status: the status of the object at the WEC.
+type Augmentation struct {
+	// `inventory` is the inventory object reference.
+	// It contains information about the inventory object (WEC).
+	Inventory InventoryObjectReference `json:"inventory"`
+	// `propagation` is the last reported metadata about one object at one WEC.
+	// It contains information about the propagation of the object.
+	Propagation PropagationMeta `json:"propagation"`
+}
+
+// InventoryObjectReference identifies an inventory object.
+// Inventory is queryable in StatusCollector expressions using the `inventory` object.
+type InventoryObjectReference struct {
+	// `name` is the inventory object's name
+	Name string `json:"name"`
+}
+
+// PropagationMeta is the last reported metadata about one workload object at one WEC.
+// PropagationMeta is queryable in StatusCollector expressions using the `propagation` object.
+type PropagationMeta struct {
+	// `infoAgeSeconds` is the age of the object in seconds.
```

Comment:
Putting an _age_ field in derived data implies the need to continually update the derived data. One reason for quantization in my original design is to limit the frequency of those recalculations. Now that I think of it, it would be better to put a timestamp than an age field in the augmentation. There could still be quantization that explicitly gives permission for a limited amount of inaccuracy in the timestamp.

Reply:
Agreed. Done.